### PR TITLE
fix: Markdownlint cleanup for RU

### PR DESCRIPTION
**Pull request by nschonni**
Part one of cleanup

### CODE REVIEW

**Comment by nschonni** [comment]
@sashasushko not sure if you have time to review this Markdownlint cleanup PRs to fix some things from the migration. Just pinging you since I saw you landed something recently 😄

**Comment by SphinxKnight** [comment]
@mdn/yari-content-ru could you please have a look?
If there is no veto or review in the 15 coming days, I'll merge this one after this time.

File: files/ru/web/html/element/image/index.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ translation_of: Web/HTML/Element/image
 
 > **Примечание:** **НИКОГДА НЕ ИСПОЛЬЗУЙТЕ ЕГО!**Для отображения изображений используйте стандартный элемент {{HTMLElement("img")}}.
 
-`Следует обратить внимание, что до Firefox 22 элемент <image>, хоть и не поддерживался, был связан с `{{domxref("HTMLSpanElement")}}. Позже это было исправлено, и теперь он связан с {{domxref("HTMLElement")}} в соответствии с требованиями спецификации.
+`Следует обратить внимание, что до Firefox 22 элемент \<image>, хоть и не поддерживался, был связан с {{domxref("HTMLSpanElement")}}. Позже это было исправлено, и теперь он связан с {{domxref("HTMLElement")}} в соответствии с требованиями спецификации.
```
Comment (sashasushko) [inline files/ru/web/html/element/image/index.md:17]:
```suggestion
Следует обратить внимание, что до Firefox 22 элемент `<image>`, хоть и не поддерживался, был связан с {{domxref("HTMLSpanElement")}}. Позже это было исправлено, и теперь он связан с {{domxref("HTMLElement")}} в соответствии с требованиями спецификации.
```

File: files/ru/web/html/element/input/range/index.md [hunk 2]
```diff
@@ -47,7 +45,7 @@ defaultValue = (rangeElem.max < rangeElem.min) ? rangeElem.min
 
 | Attribute       | Description                                                                         |
 | --------------- | ----------------------------------------------------------------------------------- |
-| [`list`](#list) | id элемента <datalist>, который содержит предопределённые значение (не обязательно) |
+| [`list`](#list) | id элемента \<datalist>, который содержит предопределённые значение (не обязательно) |
```
Comment (sashasushko) [inline files/ru/web/html/element/input/range/index.md:48]:
```suggestion
| [`list`](#list) | id элемента `<datalist>`, который содержит предопределённые значение (не обязательно) |
```

File: files/ru/web/html/element/input/search/index.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ In addition to the attributes that operate on all {{HTMLElement("input")}} eleme
 
 | Attribute                     | Description                                                                                                                         |
 | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
-| [`list`](#list)               | The id of the <datalist> element that contains the optional pre-defined autocomplete options                                        |
+| [`list`](#list)               | The id of the \<datalist> element that contains the optional pre-defined autocomplete options                                        |
```
Comment (sashasushko) [inline files/ru/web/html/element/input/search/index.md:35]:
```suggestion
| [`list`](#list)               | The id of the `<datalist>` element that contains the optional pre-defined autocomplete options                                        |
```

**Review by sashasushko — APPROVED** [review]

**Comment by nschonni** [comment]
@SphinxKnight not sure if you want to merge these now. They've gotten reviews now

**Comment by SphinxKnight** [comment]
since @sashasushko signed off on this one, let's get this merged.

**Comment by nschonni** [comment]
Thanks @SphinxKnight , #8260 and #8261 also have reviews, but need landing

**Pull request merged**